tensorflow/python/keras/layers/core.py

```diff
@@ -1021,6 +1021,8 @@ def call(self, inputs):
     rank = common_shapes.rank(inputs)
     if rank > 2:
       # Broadcasting is required for the inputs.
+      # Computing sum of products of inputs and weights -> (self.kernel) on axes -> ([[rank-1], [0]])
+      # tensordot(inputs, weights, axes)
```

Comment:
This still feels unclear to me and I think that the `tensordot` API documentation is sufficient here and we probably do not need this comment.